photon-client/src/integTest/scala/com/linkedin/photon/ml/data/avro/AvroDataReaderIntegTest.scala

```diff
@@ -125,10 +130,10 @@ object AvroDataReaderIntegTest {
   private val duplicateFeaturesPath = new Path(inputDir, "duplicateFeatures")
   private val indexMapPath = new Path(inputDir, "feature-indexes")
   private val numPartitions = 4
-  private val featureSectionMap = Map(
-    "shard1" -> Set("userFeatures", "songFeatures"),
-    "shard2" -> Set("userFeatures"),
-    "shard3" -> Set("songFeatures"))
+  private val featureShardConfigurationsMap = Map(
```

Comment:
just minor thing: will be better has a hasIntercept=false here.

Reply:
Could you elaborate - better how?

Reply:
`"shard3" -> FeatureShardConfiguration(Set("songFeatures"), hasIntercept = false` so it would test other condition tested.

Reply:
Done